mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/mapping/impl/PojoTypeManager.java

```diff
@@ -54,6 +61,15 @@ public void close() {
 		}
 	}
 
+	@Override
+	public void appendTo(ToStringTreeBuilder builder) {
+		builder.attribute( "indexedJavaClass", indexedJavaClass )
+				.attribute( "identifierMapping", identifierMapping )
+				.attribute( "routingKeyProvider", routingKeyProvider )
+				.attribute( "processor", processor )
+				.attribute( "indexManager", indexManager );
```

Comment:
Let's put the index manager at the top. It's the most global object and the processor can be multiline so the index manager ends up mostly unnoticed.

Reply:
I was thinking about putting the details of the index managers (schema) at some point in the future, which would be very verbose. But for now you're right, it makes more sense to put it at the top, I'll do that.